src/components/Header/Header.tsx

```diff
@@ -0,0 +1,27 @@
+import './Header.sass';
+import NavMenu from '@/components/NavMenu/NavMenu';
+import SubHeader from '../SubHeader/SubHeader';
+
+const Header: React.FC = () => {
+  const links = [
```

Comment:
Put it out of the components if you you static data to avoid unneeded re-renders.